runtime/doc/evim.1

```diff
@@ -28,14 +28,18 @@ See vim(1) for details about Vim, options, etc.
 The 'insertmode' option is set to be able to type text directly.
 .br
 Mappings are setup to make Copy and Paste work with the MS-Windows keys.
-CTRL-X cuts text, CTRL-C copies text and CTRL-V pastes text.
-Use CTRL-Q to obtain the original meaning of CTRL-V.
+CTRL\-X cuts text, CTRL\-C copies text and CTRL\-V pastes text.
+Use CTRL\-Q to obtain the original meaning of CTRL\-V.
```

Comment:
why those added backslashes?

Reply:
@chrisbra 
Christian, I think the keyboard shortcut in the documentation is not the same as a phrase like "long-term". It's possible that it's not.

> The hyphen-minus	is a particularly  unfortunate	case  of  overloading.
>        Its awkward name	in ISO 8859 and	later standards	reflects the many dis-
>        tinguishable  purposes  to  which it had	already	been put by the	1980s,
>        including a hyphen, a minus sign, and (alone or in  repetition)	dashes
>        of varying widths.  For best results in roff systems, use the "-" char-
>        acter  in input outside an escape sequence only to mean a hyphen, as in
>        the phrase "long-term".	For a minus sign in running  text  or  a  Unix
>        command-line option dash, use \-	(or \[-] in groff if you find it helps
>        the  clarity  of	the source document).  (Another	minus sign, for	use in
>        mathematical equations, is available as \[mi]).	AT&T  troff  supported
>        em-dashes as \(em, as does groff.

https://man.archlinux.org/man/groff_char.7.en

Reply:
Yes... My poor knowledge of English typography let me down, of course. I meant
"Unix command-line option dash" rather than a minus sign.
Thanks for the pointer. I've fixed it.